core-contracts/Delegation/src/main/java/finance/omm/score/core/delegation/DelegationImpl.java

```diff
@@ -150,13 +150,9 @@ public void addAllContributors(Address[] _preps) {
     }
 
     @External
-    public void clearPrevious(Address _user) {
```

Comment:
I believe the frontend utilizes this API, so it may be necessary to communicate any modifications to the frontend team.

Reply:
Could also be made optional and ignored